src/VirtoCommerce.CustomerModule.Data/Services/MemberService.cs

```diff
@@ -142,11 +142,15 @@ public virtual async Task SaveChangesAsync(Member[] members)
                         var dataSourceMember = AbstractTypeFactory<MemberEntity>.TryCreateInstance(memberEntityType.Name);
                         if (dataSourceMember != null)
                         {
-                           dataSourceMember.FromModel(member, pkMap);
+                            dataSourceMember.FromModel(member, pkMap);
 
                             var dataTargetMember = existingMemberEntities.FirstOrDefault(m => m.Id == member.Id);
                             if (dataTargetMember != null)
                             {
+                                /// Workaround to trigger update of audtiable fields when only updating navigation properties.
+                                /// Otherwise on update trigger is fired only when non navigation properties are updated.
+                                repository.SetChangedState(dataTargetMember);
```

Comment:
Add unit-test to this call.

Reply:
How about doing it this way?
https://github.com/VirtoCommerce/vc-module-catalog/blob/support/2.x/VirtoCommerce.CatalogModule.Data/Services/ItemServiceImpl.cs#L180

Reply:
fixed, unit test added